### DISCUSSION TITLE
Bug on laptop keyboard during gameplay

### DISCUSSION THREAD

**Discussion by DaggerBxii** [ppy/osu#17859] [Q&A] [answered]
During gameplay when playing on my laptop keyboard, there are points within the beatmap that when playing, a screenshot gets taken and my display mode changes from fullscreen to windowed (or borderless, depends on which one it's already on, but the order it goes in  is fullscreen, windowed, borderless then repeats)  while solely pressing key 1 and 2. The bug happens very inconsistently as well, but when using an external keyboard. it disappears. It may be something to do with my laptop being a touchscreen laptop. Specs and the video below: 

hp-a010dx (swapped out motherboard from a hp 15-u011dx, but they're basically the same laptop but with different specs)
Intel i7-4510U
4GB RAM
Intel HD graphics 4400
Touchscreen Laptop from around 2016

At the end of the video, you can see the screenshots taken when finished playing despite not again pressing anything other key 1 and 2, though sideways: [https://streamable.com/qkt4hj](url)

**Comment by peppy** [comment, marked as answer]
I think your wrist is probably hitting your keyboard's "Fn" key and activating F keys, then your key presses are activating F11 and F12.

Usually you can disable the Fn key via your laptop's software if you don't use it much.

**Reply by DaggerBxii** [reply]
I'm going to try and disabling it to see if it works or not

**Comment by DaggerBxii** [comment]
The problem is still occuring unfortunately. I also tried it before and after disabling on osu! stable for comparison, and it was working completely fine.

**Reply by peppy** [reply]
No idea sorry. It's going to be something on your system causing function keys to be pressed. You'll need to figure this out or contact tech support for your system.

**Reply by DaggerBxii** [reply]
I tried some setting in my bios and seemed to disappear now, so it was just a system problem.
